### PR TITLE
Adding the main page to the Community section

### PR DESCRIPTION
**Pull request by MarysiaLowas**
This pull requests adds the main page to the Community part. As an experiment, it also adds a separate menu for all jobs and meetups related pages.

### CODE REVIEW

File: jobs/urls.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from . import views
 
 urlpatterns = patterns('',
+    url(r'^$', views.main, name='main'),
```
Comment (rebkwok) [inline jobs/urls.py:6]: There currently is no /community url (in the main djangogirls/urls.py); if we're adding jobs to the community page, let's also change the root /jobs to /community (then we have e.g.  /djangogirls/community/jobs rather than /djangogirls/jobs/jobs )

File: templates/jobs/base.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        {% block content %}{% endblock %}
+    </div>
+
+    {% block footer %}
+        {% include 'global/footer.html' %}
+    {% endblock %}
+
+    {% include 'includes/js.html' %}
+</body>
+</html>
```
Comment (rebkwok) [inline templates/jobs/base.html]: This could just extend global/base and replace the menu block.  Everything else looks identical.

File: templates/jobs/main.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+<div class="container">
+    <section id="community">
+        <div class="row">
+            <div class="col-md-9">
+                <h2>Community</h2>
+
+                <p>Here, you will find information about job offers for
+                    juniors, meetups, local groups and other things that will
+                    keep you get going with becoming the fully-fledged Django
+                    programmer.</p>
```
Comment (rebkwok) [inline templates/jobs/main.html]: Suggest rewording:
Here, you will find information about job offers for entry-level positions, meetups, local groups and other things that will help you get going with becoming a fully-fledged Django programmer.
(Although I'm not sure about specifying entry-level/junior positions - will the jobs definitely be strictly limited to junior positions?)

File: templates/jobs/main.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    <section id="features">
+        <div class="row">
+            <div class="col-md-6">
+                <div class="panel panel-feature-orange">
+                    <div class="panel-heading">
+                        Jobs
+                    </div>
+                    <div class="panel-body">
+                        <p>Here you can find job offers for junior
+                            positions.</p>
```
Comment (rebkwok) [inline templates/jobs/main.html:44]: change "junior" to "entry-level" if it's changed in the description above

File: templates/jobs/main.html (hunk 3 above)
Comment (MarysiaLowas) [inline templates/jobs/main.html]: I guess we can remove _entry-level_ anytime we need. And Mikey is to review the content, so we can discuss it then. What d'you think?
